src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.html

```diff
@@ -1,14 +1,69 @@
+<xuilib-hmcts-error-summary [errorMessages]="validationErrors" [header]="'There is a problem'"></xuilib-hmcts-error-summary>
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-three-quarters">
-    <exui-hearing-parties-title></exui-hearing-parties-title>
-    <h1 class="govuk-heading-l">
-      Do you require any additional facilities?
-    </h1>
-    <exui-case-flags
-      [info]="'Any active flags on this case may require additional facilities at the hearing.'"
-      [hearingValueModel]="serviceHearingValuesModel"
-      [caseFlagsRefData]="caseFlagsRefData"
-      [caseFlagType]="caseFlagType">
-    </exui-case-flags>
-  </div>
+  <form class="form" [formGroup]="hearingFactilitiesForm">
+    <div class="govuk-grid-column-three-quarters">
+      <exui-hearing-parties-title></exui-hearing-parties-title>
+      <h1 class="govuk-heading-l">
+        Do you require any additional facilities?
+      </h1> 
+      <exui-case-flags
+        [info]="'Any active flags on this case may require additional facilities at the hearing.'"
+        [hearingValueModel]="serviceHearingValuesModel"
+        [caseFlagsRefData]="caseFlagsRefData"
+        [caseFlagType]="caseFlagType">
+      </exui-case-flags>
+    </div>
+    <div class="govuk-grid-column-two-thirds">
```

Comment:
no need to define different grid width here, can reuse line 4
<div class="govuk-grid-column-three-quarters">

Reply:
This again is the prototype design. Will reuse as stated.